scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -0,0 +1,42 @@
+:context: sno-du-enabling-workload-partitioning-on-single-node-openshift
+
+toc::[]
+
+In resource-constrained environments, such as single node production deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure OpenShift to run on a fixed number of CPUs within the host. In these environments, management workloads, including the OpenShift control plane, need to be configured to
+use fewer resources than they might by default in normal clusters.
+
+
+You can isolate the {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. This is useful for resource-constrained environments, such as a single node cluster, where you want to reserve most of the CPU resources for user workloads and configure {product-title} to run on a fixed number of CPUs within the host.
+
+When you use workload partitioning, the CPU resources used by {product-title} for cluster management are isolated to a partitioned set of CPU resources on a single node cluster. This partitioning isolates cluster management functions to the defined number of CPUs. All cluster management functions operate solely on that `cpuset`.
+
+The minimum number of reserved CPUs required for the management partition for a single node cluster is four CPU Hyper threads (HTs). Inclusion of Operators or workloads outside of the set of accepted management pods requires
+additional CPU HTs.
+
+The set of pods that make up the baseline OpenShift installation and a set of typical addon operators are annotated for workload partitioning by default. These pods normally operate within the minimum size `cpuset`. Any additional pods annotated for inclusion in the management workload partition require that additional CPU resources be added to the partition.
+
+Workload partitioning isolates the workloads away from the platform workloads using the normal scheduling capabilities of Kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing cluster management workloads and user workloads.
+
+To fully leverage workload partitioning, you must install the Performance Addon Operator and apply the performance profile:
+
+* Workload partitioning pins the {product-title} infrastructure pods to a defined `cpuset`.
+* The Performance Addon Operator performance profile pins the os/systemd services to a defined `cpuset`.
+* This `cpuset` should match.
+
+The concept of cluster management workloads is flexible and can encompass:
+
+* All {product-title} core components necessary to run the cluster.
+
+* Any add-on Operators necessary to make up the platform as defined by the customer.
+
+* Operators or other components from third-party vendors that the customer deems as management rather than operational.
+
+** For the DU RAN use case, a designated set of Operators is defined. Anything beyond that requires increasing the default number of CPUs for workload partitioning.
```

Comment:
I still think that this list implies too much flexibility for the customer. For the "management" workload partition the set of content is pre-defined by us and if the user adds anything to it they would need to increase the number of cores to support it. I would remove lines 30-38 and add to lines 17-18 the following text:

> The set of pods which make up the baseline OpenShift installation and a set of typical add-on operators are annotated for inclusion in the management workload partition. These pods will operate normally within the minimum size cpuset. Any additional pods annotated for inclusion in the management workload partition will require that additional CPU resources be added to that partition.